Source/Pawnmorphs/Esoteria/FormerHumanUtilities.cs

```diff
@@ -123,15 +123,74 @@ static FormerHumanUtilities()
 
             Giver_RecruitSapientAnimal.ResetStaticData();
 
+            _allRegularFormerHumanPawnKinds = new List<PawnKindDef>();
+
+            if (PawnmorpherMod.Settings.animalBlacklist == null)
+                PawnmorpherMod.Settings.animalBlacklist = GetDefaultBlockList();
+
+			CacheValidFormerHumans();
+		}
+
+		/// <summary>
+		/// Determines whether [is valid former human] [the specified forced].
+		/// </summary>
+		/// <param name="pawnKind">Kind of the pawn.</param>
+        /// <param name="allowDisabled">Include disabled animals.</param>
+        /// <param name="allowRestricted">Include restricted animals.</param>
+		/// <returns>
+		///   <c>true</c> if [is valid former human] [the specified forced]; otherwise, <c>false</c>.
+		/// </returns>
+		public static bool IsValidFormerHuman(this ThingDef pawnKind, bool allowRestricted = true, bool allowDisabled = false)
```

Comment:
What was the reason for defaulting allowRestricted to true?

Reply:
Even restricted, they are technically still valid former humans.
But it is entirely semantic.